test/lib/ansible_test/_data/sanity/validate-modules/validate_modules/main.py

```diff
@@ -918,6 +948,7 @@ def _validate_docs(self):
                     # for error in errors:
                     #     self.reporter.error(
                     #         path=self.object_path,
+                    #         name='ansible-metadata-format',
```

Comment:
Was the YAML feature removed from the spec? If so, this whole commented block of code can be removed in a separate PR.